extension/commands.ts

```diff
@@ -33,6 +33,8 @@ export class UefiCommands implements vscode.Disposable {
     dispose() {
     }
 
+    Lets add a change here.
```

Comment:
I don't like this change.

Reply:
This didn't show up.  @matthewfcarlson  needs to review how he handles PR review comments.

Reply:
I'm working on it :)

Reply:
Here we go :)

Reply:
There we go :)

Reply:
This is working